<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PinInfoWidget</class>
 <widget class="QWidget" name="PinInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>403</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑 Light</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Pin configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0">
    <widget class="QLabel" name="label_devchnl">
     <property name="text">
      <string>设备通道</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QComboBox" name="comboBox_devchnl"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_devfunc">
     <property name="text">
      <string>外设类型</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="comboBox_rttport"/>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QComboBox" name="comboBox_vol5v">
     <item>
      <property name="text">
       <string>Yes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="comboBox_rttpin"/>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QPushButton" name="pushButton_save">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QComboBox" name="comboBox_usefunc"/>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QComboBox" name="comboBox_devname"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_devname">
     <property name="text">
      <string>设备名字</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_vol5v">
     <property name="text">
      <string>5V容忍</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QComboBox" name="comboBox_devfunc"/>
   </item>
   <item row="0" column="0" colspan="3" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label_title">
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_usefunc">
     <property name="text">
      <string>引脚功能</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_rttportpin">
     <property name="text">
      <string>引脚编号</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
